src/FilterConditionSelector/FilterConditionSelector.tsx

```diff
@@ -47,11 +47,11 @@ const FilterConditionSelector: React.FC<
 	const mainRef = React.useRef<HTMLElement | null>(null);
 	const [searchTerm, setSearchTerm] = React.useState(props.searchTerm);
 	const [
-		firstItemTree,
-		setFirstItemTree
+		treeFirstItem,
+		setTreeFirstItem
 	] = React.useState<HTMLDivElement | null>(null);
 	const debouncedSearchTerm = useDebounce(searchTerm, 500);
-	let markedFirstItemTree: () => void;
+	let markFirstItemTree: () => void;
 	let unmarkedFirstItemTree: () => void;
```

Comment:
Ну строчкой ниже тоже надо было посмотреть
```suggestion
	let unmarkFirstItemTree: () => void;
```